package.json

```diff
@@ -9,8 +9,12 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@chakra-ui/react": "^2.4.6",
+    "@emotion/react": "^11",
+    "@emotion/styled": "^11",
     "@rainbow-me/rainbowkit": "^0.8.1",
     "ethers": "^5.0.0",
+    "framer-motion": "^6",
     "next": "^12.1.6",
```

Comment:
Should we use next.js v13? That's the default used when I scaffold a new project, with create-next-app or create-wagmi, so wondering why we're on v12

Reply:
Not sure, I used `yarn create @rainbow-me/rainbowkit@latest`. I can try upgrading to 13

Reply:
Feel free to create an issue and handle in a separate PR, small PRs are nice 🙂

Reply:
Created and issue #6